core/db.py

```
@@ -106,7 +107,7 @@ def execute(self, statement: Any, *args: Any, **kwargs: Any):
             result = conn.execute(stmt, *args, **kwargs)
             try:
                 conn.commit()
-            except Exception:
+            except Exception:  # nosec B110
                 # Not all statements require/allow commit; ignore commit errors
                 pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)